src/org/usfirst/frc2175/subsystem/intake/DreamIntakeSubsystem.java

```diff
@@ -11,12 +12,41 @@
     private Talon dreamIntakeMainBeltTalon;
     private Talon dreamIntakeLiftTalon;
 
+    private double sideBeltSpeedForward;
+    private double sideBeltSpeedReverse;
+    private double mainBeltSpeedForward;
+    private double mainBeltSpeedReverse;
+
     public DreamIntakeSubsystem(RobotConfig robotConfig) {
         WiringConfig wiringConfig = robotConfig.getWiringConfig();
 
         dreamIntakeSideBeltTalon = wiringConfig.getDreamIntakeSideBeltTalon();
         dreamIntakeMainBeltTalon = wiringConfig.getDreamIntakeMainBeltTalon();
         dreamIntakeLiftTalon = wiringConfig.getDreamIntakeLiftTalon();
+
+        IntakeConfig intakeConfig = robotConfig.getIntakeConfig();
+
+        sideBeltSpeedForward = intakeConfig.getSideBeltSpeedForward();
+        sideBeltSpeedReverse = intakeConfig.getSideBeltSpeedReverse();
+        mainBeltSpeedForward = intakeConfig.getMainBeltSpeedForward();
+        mainBeltSpeedReverse = intakeConfig.getMainBeltSpeedReverse();
+
+    }
+
+    public double getSideBeltSpeedForward() {
+        return sideBeltSpeedForward;
+    }
+
+    public double getSideBeltSpeedReverse() {
+        return sideBeltSpeedReverse;
+    }
+
+    public double getMainBeltSpeedForward() {
+        return mainBeltSpeedForward;
+    }
+
+    public double getMainBeltSpeedReverse() {
+        return mainBeltSpeedReverse;
```

Comment:
You actually don't need to have any of this in `DreamIntakeSubsystem`...you only need to load these values out of `IntakeConfig` in the specific commands. So you don't need these getter methods or the variables that they are returning.